### PR TITLE
Feature add jigsaw

### PR DESCRIPTION
**Pull request by FrancisBehnen**

### CODE REVIEW

**Comment by FrancisBehnen** [comment]
Logo looks horrible. Working on it.

File: scripts/unofficial-tournaments.json [hunk 1]
```diff
@@ -213,5 +213,45 @@
       "Kali Linux"
     ],
     "footer": false
+  },
+  {
+      "title": "8-Bits Retro-Gaming",
+      "subtitle": "",
+      "headerTitle": "8-Bits Retro-Gaming",
+      "description": "This 8-bits retro gaming competition will take place the whole weekend. We'll be playing a classic on an old Atari game console. If you submit your score to the organisation you take place in the competition and make a chance to win one of the awesome eight prizes.",
```
Comment (skamoen) [inline scripts/unofficial-tournaments.json]: "make a chance" is not English

File: scripts/unofficial-tournaments.json [hunk 2]
```diff
@@ -213,5 +213,45 @@
...
+      },
+      "prizes": [
+        "Asus Strix Tactic Pro Keyboard",
+        "Logitec G G430",
+        "Mega Bombox Touchspeaker",
+        "RockStar hoofdtelefoonsplitser",
+        "Schermreiniging",
+        "Touchscreen handschoenen",
+        "Biertje en Snicker",
+        "Biertje"
```
Comment (skamoen) [inline scripts/unofficial-tournaments.json]: This is not English either

Comment (FrancisBehnen) [inline scripts/unofficial-tournaments.json]: Ohw yea, lol

**Pull request closed without merging**